main.py

```diff
@@ -374,9 +374,14 @@ def get(self, student_name):
         firstname = student_names[0].lower()
         lastname = student_names[1].lower()
         student_schedule = self.get_schedule_for_name(firstname, lastname)
+
         user_schedule = self.get_schedule_for_id(id)
 
         sanitized = self.sanitize_schedule(student_schedule, user_schedule)
+
+        # Generate email address
+        sanitized["email"] = firstname[0] + lastname.replace(" ", "") + "@eastsideprep.org"
```

Comment:
This seems like it should be a make_email_address(firstname, lastname) function. If the format ever changes, you only have to update one place.